src/components/Assets/Assets.js

```diff
@@ -0,0 +1,246 @@
+import React, { Component } from "react";
+import AssetTable from "./AssetTable";
+import AssetMap from "./AssetMap";
+import AssetDetailsSection from "./AssetDetailsSection";
+import { FIELDS } from "./fieldConfig";
+import { getAllAssets } from "../WorkOrder/helpers";
+import { getAllAssetDetails, formatDataTitles } from "./helpers";
+
+import Autocomplete from "react-autocomplete";
+import changeCase from "change-case";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faMapMarkerAlt,
+  faSpinner,
+  faInfoCircle,
+} from "@fortawesome/free-solid-svg-icons";
+import {
+  Accordion,
+  AccordionItem,
+  AccordionItemTitle,
+  AccordionItemBody,
+} from "react-accessible-accordion";
+import "react-accessible-accordion/dist/fancy-example.css";
+
+class Assets extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      assetsData: [],
+      assetOptions: [],
+      loading: false,
+      location: "",
+      typedAsset: "",
+      selectedAsset: "",
+      assetDetailsData: "",
+      assetServiceRequestsData: "",
+      assetDetectorsData: "",
+      assetPreventativeMaintenanceData: "",
+      assetWorkOrdersData: "",
+      assetMapData: "",
+      assetCamerasData: "",
+      assetSignalPriorityData: "",
+      assetPoleAttachmentsData: "",
+      assetTravelSensorData: "",
+      assetApsButtonRequestsData: "",
+      assetCadStatusData: "",
```

Comment:
so much data... 🤯

Reply:
Yeah - all of the field mapping had me going 😮